readme.md

```diff
@@ -77,13 +81,53 @@ Where:
     generated via the implementation. See the `Zip format` section
     for more information about its structure.
 
-## Zip format (R2)
+## Zip Format (R3)
+
+### Certificates - artifacts_certs_r3.zip
+
+Starting with artifacts for the NIST Draft standards released 2023-08-24, we will use a much simpler artifact format:
+
+* Only produce a self-signed certificate (TAs). Let's not bother with CA / EE / CRL / OCSP; those are begging for compatibility issues that have nothing to do with the PQ algs.
+* We will restrict the R3 artifacts to only the algorithms with NIST draft standards.
+* Use PEM formats.
+* Switch to a flat folder structure with filenames <oid>_ta.pem
+* For Kyber, use the the Dilithium TA of the equivalent security level to sign a <kyber_oid>_ee.pem
+
+Within `providers/<provider_name>/`
+- artifacts_certs_r3.zip
```

Comment:
This doesn't match the OID mapping table here:  https://github.com/IETF-Hackathon/pqc-certificates/blob/master/docs/oid_mapping.md